security-force-monitor/sfm-cms
# Add attribution to unit maps, update translation files

## Overview

See title.

Connects #772, external issue.

## Testing Instructions

* Deploy to staging
* View the Units referenced in #772 and confirm the site and area maps have appropriate attribution
